packages/models-library/src/models_library/workspaces.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+from typing import TypeAlias
+
+from pydantic import BaseModel, Field, PositiveInt
+
+WorkspaceID: TypeAlias = PositiveInt
+
+
+#
+# DB
+#
+
+
+class WorkspaceDB(BaseModel):
+    workspace_id: WorkspaceID
+    name: str
+    description: str | None
+    owner_primary_gid: PositiveInt = Field(
+        ...,
+        description="GID of the group that owns this wallet",
+    )
+    thumbnail: str | None
+    created: datetime = Field(
+        ...,
+        description="Timestamp on creation",
+    )
+    modified: datetime = Field(
+        ...,
+        description="Timestamp of last modification",
+    )
+
+    class Config:
+        orm_mode = True
+
+
+class UserWorkspaceDB(WorkspaceDB):
```

Comment:
User or Group?

Reply:
renamed -> UserWorkspaceAccessRightsDB its a combination of Workspace + access rights for a user, used only in query when you want workspace access rights for a user